homeassistant/components/binary_sensor/rpi_gpio.py

```diff
@@ -49,30 +51,32 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     ports = config.get('ports')
     for port_num, port_name in ports.items():
         binary_sensors.append(RPiGPIOBinarySensor(
-            port_name, port_num, pull_mode, bouncetime, invert_logic))
+            hass, port_name, port_num, pull_mode, bouncetime, invert_logic))
     add_entities(binary_sensors, True)
 
 
 class RPiGPIOBinarySensor(BinarySensorDevice):
     """Represent a binary sensor that uses Raspberry Pi GPIO."""
 
-    def __init__(self, name, port, pull_mode, bouncetime, invert_logic):
+    def __init__(self, hass, name, port, pull_mode, bouncetime, invert_logic):
```

Comment:
Don't pass in `hass`. It will be set on the entity when it has been added to home assistant.